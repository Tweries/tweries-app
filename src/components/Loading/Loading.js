import React from 'react';

const copy = {
  '...': '...'
};

function Loading() {
  return (
    <article className="container content-center mx-auto m-1 p-4 text-center">
      {copy['...']}
    </article>
  );
}

export default Loading;
