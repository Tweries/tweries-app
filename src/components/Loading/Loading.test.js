import { render } from '@testing-library/react';
import React from 'react';
import Loading from './Loading';

describe('<Loading />', () => {
  it('to match snapshot', () => {
    const { container } = render(<Loading />);
    expect(container).toMatchSnapshot();
  });
});
