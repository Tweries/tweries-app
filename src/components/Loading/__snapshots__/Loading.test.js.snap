// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> to match snapshot 1`] = `
<div>
  <article
    class="container content-center mx-auto m-1 p-4 text-center"
  >
    ...
  </article>
</div>
`;
